collect.py

```diff
@@ -203,6 +203,19 @@ def screenshot(driver, file_path):
     image.save(file_path)
 
 
+def get_code(driver, file_path):
+    if(not(os.path.isdir('source'))):
+        utils.mkdir('source')
+    source = driver.page_source
+    with open(file_path, "w", encoding="utf-8") as open_file:
+        open_file.append("source code : " + source + "\n")
+
+
+def get_screenshot_code(driver, location):
+    screenshot(driver, 'data/' + location + '.png')
+    get_code(driver, 'source/' + location + '.txt')
```

Comment:
Store everything in the same directory **data**. The name of the file containing the DOM tree can be called dom_LOCATION

Also change the **location** name to something more meaningful.